Cython/Build/IpythonMagic.py

```diff
@@ -243,6 +243,10 @@ def f(x):
         help=("Print debug information like generated .c/.cpp file location "
               "and exact gcc/g++ command invoked.")
     )
+    @magic_arguments.argument(
+        '--show-cythonized-code', action='store_true', default=False,
+        help="Show cythonized c/cpp file (ignored if in combination with -a/--annotate)."
```

Comment:
Silently ignoring one seems surprising, especially since both result in "HTML" output.